linux_os/guide/system/accounts/accounts-session/accounts_users_netrc_file_permissions/oval/shared.xml

```diff
@@ -0,0 +1,49 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Netrc User File In The Home Directory Must Not be group or world
+    accessible") }}}
+    <criteria>
+      <criterion test_ref="test_accounts_users_home_netrc_file_permissions"
+                 comment="Netrc User File In The Home Directory Must Not be group or world
+                          accessible"/>
+    </criteria>
+  </definition>
+
+  <unix:password_object id="object_accounts_users_home_netrc_file_permissions_objects" version="1">
+    <unix:username datatype="string" operation="pattern match">.*</unix:username>
+    <filter action="include">state_{{{ rule_id }}}_interactive_uids</filter>
+  </unix:password_object>
+
+  <unix:password_state id="state_{{{ rule_id }}}_interactive_uids"
+                       version="1">
+    <unix:user_id datatype="int" operation="greater than or equal">{{{ uid_min }}}</unix:user_id>
+  </unix:password_state>
```

Comment:
Recently it was made a work in the project aiming to align the approach to identify interactive users:
https://github.com/ComplianceAsCode/content/pull/10215

It would be great to keep this rule also consistent with the others. Here is an example:
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-session/accounts_user_dot_no_world_writable_programs/oval/shared.xml#L10